pdns/sdig.cc

```diff
@@ -27,6 +27,19 @@ void usage() {
   cerr<<"Syntax: sdig IP-ADDRESS PORT QUESTION QUESTION-TYPE [dnssec] [recurse] [showflags] [hidesoadetails] [hidettl] [tcp] [ednssubnet SUBNET/MASK] [xpf XPFDATA]"<<endl;
 }
 
+const string nameForClass(uint16_t qclass, uint16_t qtype)
```

Comment:
perhaps we need a `QClass::toString()` so we don't need this function

Reply:
Yes, makes sense, and then we can also fix this in the other tools.